### PR TITLE
Maybe smaller binary

### PR DESCRIPTION
**Pull request by alexey-milovidov**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: src/Functions/wkt.cpp [hunk 1]
```diff
@@ -52,6 +55,7 @@ class FunctionWkt : public IFunction
             for (size_t i = 0; i < input_rows_count; ++i)
             {
                 std::stringstream str; // STYLE_CHECK_ALLOW_STD_STRING_STREAM
+                str.exceptions(std::ios::failbit);
```
Comment (alexey-milovidov) [inline src/Functions/wkt.cpp:58]: Here was a bug.

**Comment by robot-clickhouse** [comment]
*This is an automated comment for commit 0de51d2b0f1d1ea3d4910b96d729ae444f6b79d7 with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/56112/0de51d2b0f1d1ea3d4910b96d729ae444f6b79d7/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td><a href="https://github.com/ClickHouse/ClickHouse/actions/runs/6687562707">✅ success</a></td></tr>
<tbody>
</table>
</details>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/56112/0de51d2b0f1d1ea3d4910b96d729ae444f6b79d7/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

**Comment by alexey-milovidov** [comment]
-5.7 MB

File: src/Functions/wkt.cpp (hunk 1 above)
Comment (Algunenano) [reply src/Functions/wkt.cpp:58]: There are quite a few other places using stringstream's without setting the exceptions bit (just look for `STYLE_CHECK_ALLOW_STD_STRING_STREAM`. Should all of them set it?

**Review by Algunenano — APPROVED** [review]

File: src/Functions/wkt.cpp (hunk 1 above)
Comment (alexey-milovidov) [reply src/Functions/wkt.cpp:58]: Yes, all places should have it. Otherwise, any exception, e.g. bad_alloc, will lead to a silent loss of data.

**Pull request merged**